app/utils/accountHelpers.ts

```diff
@@ -163,16 +163,16 @@ export function getPublicAccountAmounts(
     if (!accountInfo) {
         return { total: 0n, staked: 0n, scheduled: 0n, atDisposal: 0n };
     }
-    const total = BigInt(accountInfo.accountAmount);
+    const total = BigInt(accountInfo.accountAmount.microCcdAmount);
     const staked =
         (accountInfo as AccountInfoBaker).accountBaker?.stakedAmount ??
         (accountInfo as AccountInfoDelegator).accountDelegation?.stakedAmount ??
```

Comment:
```suggestion
        (accountInfo as AccountInfoBaker).accountBaker?.stakedAmount.microCcdAmount ??
        (accountInfo as AccountInfoDelegator).accountDelegation?.stakedAmount.microCcdAmount ??
```

Reply:
yeah.. thats pretty weird it did not complain.. good catch!